project/iterative_sorting.py

```diff
@@ -23,7 +23,13 @@ def selection_sort( arr ):
 
 # TO-DO: implement the Insertion Sort function below
 def insertion_sort( arr ):
-
+  for i in range(1, arr(len)):
+    current = arr[i]
+    j= i - 1
+    while j >= 0 and current <= j:
```

Comment:
current is an element in the array, and you are comparing it to the index value you're iterating over.